src/include/daos_errno.h

```diff
@@ -277,7 +277,16 @@ extern "C" {
 	       One or more control plane components are incompatible)	\
 	/** No service available */					\
 	ACTION(DER_NO_SERVICE,		(DER_ERR_DAOS_BASE + 39),	\
-	       No service available)
+	       No service available)					\
+	/** Requested operation failed */				\
+	ACTION(DER_OP_FAILED,		(DER_ERR_DAOS_BASE + 40),	\
+	       Requested operation failed)				\
+	/** Unable to verify operation result */			\
+	ACTION(DER_OP_NOT_VERIFIED,	(DER_ERR_DAOS_BASE + 41),	\
+	       Cannot verify operation result)				\
+	/** Unexpected operation result */				\
+	ACTION(DER_OP_UNEXP_RESULT,	(DER_ERR_DAOS_BASE + 42),	\
+	       Unexpected operation result)
```

Comment:
Is this PR/feature of big enough magnitude that we need to add new error codes for it?

Reply:
ref: https://github.com/daos-stack/daos/pull/9063#discussion_r882102750

Reply:
I agree with @ashleypittman , it's not necessary to introduce new error codes which are only meaningful for these specific LED operations, unless they could be applied to other generic use cases.

Reply:
The question for me was that I believe these errors are being returned all the way up to the dmg tool and being displayed to the caller. In my view there's potential for confusion if the command succeeds but the internal verification fails for some reason or another.

Reply:
The command obviously shouldn't succeed if the internal verification fails for any reason, that's not what I'm suggesting.  Appropriate error messages are important, but we do not want a unique error code for every possible fault as the surface for that would be so big.

Reply:
done

Reply:
Sorry if I'm being unclear, I mean it is confusing to return an error if the command succeeded but we failed to verify one way or the other. I'm thinking of something like:

- User requests LED change
- set_led succeeds
- get_led fails
- User sees "function not implemented" (DER_NOSYS)

The LED is probably on. The original command succeeded. The user reading this message may reasonably believe the LED is not on, because of something not being implemented. Which is, at best, misleading. We don't actually have good error messages for "couldn't verify," which is why I suggested adding generic ones.

I was talking with Tom about this offline, and we agreed to revert for now just to get the SPDK update landed. But I think we will probably need to revisit this.